### PR TITLE
Convert to a monorepo

### PR DESCRIPTION
**Pull request by daniel-ac-martin**
Converts to a monorepo (using pnpm) and adds new packages for consuming the components.

## Known regressions

- Storybook's accessibility addon is broken (wait for stable update from Storybook)
- Storyshots is not implemented (no value?)
- Storyshots-puppeteer is not implemented (seems to be buggy, so might not be worth implementing)
- The static storybook is broken and missing at least its `index.html` file.
- Storybook notes is gone (seems to be deprecated - remove from `.stories.mdx` files?)

### CODE REVIEW

File: netlify.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+# Build docs website
+cd ./apps/govuk-docs/
+npm run build
+
+# Package functions
+../../node_modules/.bin/sls package
+echo "process.env['MODE'] = 'serverless'; module.exports = require('./dist/server/index.js');" > govuk-docs.js
+zip -rv .serverless/govuk-docs.zip govuk-docs.js
+rm govuk-docs.js
```
Comment (RobinKnipe) [inline netlify.sh:19]: May I suggest:
```shell
echo "process.env['MODE'] = 'serverless'; module.exports = require('./dist/server/index.js');" | zip -v .serverless/govuk-docs -
```
...in place of the above 3 lines?

Comment (RobinKnipe) [reply netlify.sh:19]: ...actually, sadly the filename comes out as `-` which probably isn't desirable. How about just delete the target file with the `-m` flag instead (of the zip/rm lines):
```shell
zip -mv .serverless/govuk-docs.zip govuk-docs.js
```

**Comment by daniel-ac-martin** [comment]
I think this is now good enough to merge as the regressions seem quite minor especially when compared to the progress made.

**Comment by daniel-ac-martin** [comment]
A `pre-monorepo` tag has been pushed to allow one to return to a point before the merging of this PR.

**Pull request merged**